react-list.es6

```diff
@@ -113,7 +116,14 @@ module.exports = class ReactList extends Component {
       }, 0);
     }
 
-    this.updateFrame();
+    if (type === 'simple' && stableFrameDelay) {
```

Comment:
Well, based on my interpretation of the documentation, on `variable` you have to provide the height, so animating your elements should not interfere with it. 
About `uniform`, I've tried to enable it on but it didn't work out of the box. So, if you think it might be useful I might give it a second try (but I might need to touch more code).